From e815bf1282b580ceeb50ce505502221d27168999 Mon Sep 17 00:00:00 2001
From: ahs <amrita.h.s@intel.com>
Date: Mon, 19 Oct 2020 10:56:18 +0530
Subject: [PATCH 1/2] Revert "Clang infers -mavx functionality in -mavx2.
 Removed -mavx flag"

This reverts commit 315c8ad00790c41fd4cefaf7f8dc7830af0f4eed.
---
 nn/Android.bp | 10 ++--------
 1 file changed, 2 insertions(+), 8 deletions(-)

diff --git a/nn/Android.bp b/nn/Android.bp
index 904718a..5b07b05 100644
--- a/nn/Android.bp
+++ b/nn/Android.bp
@@ -42,18 +42,12 @@ cc_defaults {
     arch: {
         x86: {
             avx2: {
-                cflags: [
-                    "-mavx2",
-                    "-mfma",
-                ],
+                cflags: ["-mavx","-mavx2","-mfma"],
             },
         },
         x86_64: {
             avx2: {
-                cflags: [
-                    "-mavx2",
-                    "-mfma",
-                ],
+                cflags: ["-mavx","-mavx2","-mfma"],
             },
         },
     },
-- 
2.7.4

